source/ota.c

```diff
@@ -547,8 +548,8 @@ static OtaStateTableEntry_t otaTransitionTable[] =
     { OtaAgentStateAll,                 OtaAgentEventShutdown,            shutdownHandler,        OtaAgentStateStopped             },
 };
 
-/* MISRA rule 2.2 warns about unused variables. These 2 variables are used in log messages, which is
- * disabled when running static analysis. So it's a false positive. */
+/* MISRA Ref 2.2.1 [Unions and dead code] */
```

Comment:
Is the comment (`[Unions and dead code]`) here correct?
I do not see unions here.